doc/guides/triaging-for-lts-releases.md

```diff
@@ -0,0 +1,127 @@
+# Triaging for LTS Releases
+
+### Set up your repo
+
+- You will need collaborator access.
+
+- Make sure nobody else is currently backporting to the same LTS branch.
+
+- Clone node
+
+  ```bash
+  git clone git@github.com/nodejs/node --origin upstream
+  ```
+
+- Make sure you're up to date with staging branch (where N is an LTS release e.g. 6,8,10)
+  ```bash
+  git fetch --all
+  git checkout -B vN.x-staging upstream/vN.x-staging
+  ```
+
+- Run the tests on the  to make sure everything works:
+  ```bash
+  ./configure && make -j4 test
+  ```
+
+- Install branch-diff (and set up the key)
+  ```bash
+  npm install -g branch-diff
+  ```
+
+### Triage PRs
+
+Go through PRs with the label `vN.x`. e.g. [PRs with the v8.x label](https://github.com/nodejs/node/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Av8.x)
+
+For each PR:
+
+- Run or check that there is a passing CI
+- Check approvals (you can approve yourself)
+- Check SemVer
+- Check the PR matches the wasn't changed from `master` commit
+- If there are merge conflicts, comment in the issue asking the PR author to rebase.
+- Land if okay
+  - Add git alias if you haven't already:
+  ```bash
+  git config --global alias.pa '!curl -L $1.patch | git am -3 -S --whitespace=fix #'
+  ```
+  - Patch Pull Request onto staging branch:
+  ```bash
+  git pa https://github.com/nodejs/node/pull/12345
```

Comment:
For ease of understanding it might be helpful to add that this patch flow also applies to commits, and add a section to commit triaging that includes an example similar to:
`$ git pa https://github.com/nodejs/node/commit/1a2b3c4d`